packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -739,9 +768,13 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
                 >
                     <span dangerouslySetInnerHTML={{__html: glyphTransparency}}/>
                 </button>
+                {this.renderMeasureSpan(this.themeRuler, `${this.state.theme.displayName}`)}
                 {this.renderThemeSelect()}
+                {this.renderMeasureSpan(this.localeRuler, `${this.state.locale}`)}
```

Comment:
backticks unnecessary since you are not concatenating anything within, see also 771